circuits/cpp/src/aztec3/circuits/apps/.test.cpp

```diff
@@ -143,12 +143,12 @@ TEST_F(state_var_tests, mapping)
 
     my_mapping[5] = to_ct(composer, NT::fr(5));
 
-    info("my_mapping[5]: ", my_mapping[5]);
-    info("my_mapping[5].start_slot: ", my_mapping[5].start_slot);
-    info("my_mapping[5].storage_slot_point: ", my_mapping[5].storage_slot_point);
+    // info("my_mapping[5]: ", my_mapping[5]);
```

Comment:
Ik youve left a note in this pr youd like to change these to debugs. But how will we stop someone from doing some spring cleaning. Should we add some notes?

Reply:
I don't think it's the end of the world if someone removes them. But I was kind of explaining away my own laziness since I think it would be better to move them into a `debug()`. Created an issue: https://github.com/AztecProtocol/aztec3-circuits/issues/248